src/posts/week-38-2020-commitment-before-accuracy.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Week 38, 2020: Commitment before accuracy"
+date: 2020-09-23
+slug: "/week-38-2020-commitment-before-accuracy"
+description: >-
+  Consider committing a direction before worrying about the accuracy of it. You can always changes paths later.
+hasTweetEmbed: false
+---
+
+## On life and writing
+
+It's been perhaps the hardest week on the writing front for some time. I’ve spent the week writing and rewriting the same piece, trying to cajole it into something I’m happy with.
+As I write one of the final drafts of this week’s review, I’m relearning some lessons all over again:
+
+I won’t always like what I write. It’s not an issue. It will be frustrating. And it’s a fact. If the day comes where I do, then I’m probably letting my standards slip, rather than writing anything of value.
+
+I don’t always have to publish what I write. It would be great if everything was up to scratch, but it won’t always be the case. My commitment is to publish something at least once a week, but why publish something I’m unhappy with? Better to chalk it up as a loss and keep moving. The most important part is that I show up and do the work.
+
+## On goal setting
+
+Over the last couple of weeks I’ve been talking about goals with various people. Member of my team at TransferWise, people I’m close with. People who are unsure of what they want from the future, or how to get there.
+
+Goal setting should be easy. The hard part should be achieving them. Goals needn’t be complicated. At best, they’re a simple tool to get you moving in the right direction.
+
+To understand how to use them as a tool, first you must start at the end. Interviews love the question “where do you see yourself in five years”? It’s not much of an interview question because it’s too easy to game. But it is a great opportunity for self reflection.
```

Comment:
```suggestion
To understand how to use them as a tool, first you must start at the end. Interviewers love the question “where do you see yourself in five years”? It’s not much of an interview question because it’s too easy to game. It is however, a great opportunity for self reflection.
```